proto/substrait/algebra.proto

```diff
@@ -131,14 +131,38 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message TextReadOptions {
```

Comment:
I think I prefer CSV / DSV / TSV over "Text".  Text is pretty commonly used in the file format domain to mean "CSV or JSON or YAML or XML".

Reply:
Changed to DelimiterSeperatedText (instead of just text).